ig/input/fsh/VacationPlan.fsh

```diff
@@ -6,9 +6,22 @@ Description: """
   VacationPlan models the intent of a patient to travel to a different location, potentially leading to a change of the
   indicated immunizations for the patient.
 """
-* extension contains VPVacationPlanExtension named vacationPlanExtension 1..1
+* code 1..1 MS
+* extension contains VPVacationPlanExtension named vacationPlanExtension 1..1 MS
 
 Extension: VPVacationPlanExtension
 Id: vp-vacation-plan-extension
 Title: "VacationPlanExtension"
 Description: ""
+* extension contains VPLocationExtension named locations 1.. MS
```

Comment:
```suggestion
* extension contains VPLocationExtension named location 1.. MS
```
I would suggest we keep this naming singular to be consistent with the other extensions. It can occur more than once but each extension is a `location`.

Reply:
Addressed at [6f667ff](https://github.com/FHIR-Vaccination-Pass/fhir-implementation-guide/pull/37/commits/6f667ff139cfdc60a2f9b5a156763bb685ba0c7e)